README.md

```diff
@@ -1,3 +1,10 @@
+# dslib thread-safe version
+
+For now the Thread-Safe mode is implemented only for avl-tree
+But in common.h and common.c there are the lock funx so is very easy extend this property at all other structure
+
+
```

Comment:
Why these extra lines?

Reply:
you told me to modify the readme, so i described actual state of code